index.html

```diff
@@ -50,7 +50,7 @@
         <li><a href="#programs">Programs</a></li>
         <li class="toggle"><a href="#main-menu">Contribute<span class="fas fa-chevron-down"></span></a>
             <ul>
-              <li><a href="/newcomers">Newcomers</a></li>
+              <li><a href="./newcomers.html">Newcomers</a></li>
```

Comment:
@simsausaurabh Unrelated change. But that's okay.

Reply:
@shreyateeza Without these changes, it was not migrating to newcomers.html locally when I was running index.html on my browser. So, I made these changes now it will run both locally and on hosted/gh-pages.

Reply:
Alright!

Reply:
Are you using ecstatic (as explained in the readme) to view the website locally?  If you are, this change shouldn't be necessary.